### PR TITLE
Major revision based primarily on feedback from Erin Robinson.

### PR DESCRIPTION
**Pull request by gvwilson**
1.  Reorder sections.
2.  Split governance and licensing material.
3.  Include more material on consensus-based governance.
4.  Add rule on evaluation.

### CODE REVIEW

File: revised.tex [hunk 1]
```diff
@@ -0,0 +1,555 @@
...
+and the citizens who support them financially and politically
+are aware of each other's efforts,
+collaborate in ad hoc ways,
+and are conscious of contributing toward a shared purpose.
+
+Open source software projects are also communities of effort.
+For example,
+the Mozilla Firefox~\cite{mozilla} community includes a mix of paid professionals,
+highly-involved volunteers,
+and occasional contributors who create software,
```
Comment (batpigandme) [inline revised.tex]: I'd give this either a _both-and_ construct or a _not only-but_ construct.
> who not only create software, documentation, and tutorials, but also…

or

> who create both software, documentation, and tutorials, as well as organize events…

File: revised.tex [hunk 2]
```diff
@@ -0,0 +1,555 @@
...
+This information should be easily accessible and guide new members to the tasks they may handle.
+LibreOffice,
+for example,
+provides a way for developers to filter available tasks by required skills and difficulty~\cite{libreoffice-filtered}.
+
+The project should also help developers evaluate their skills,
+since ``basic Python skills'' means very different things to different people.
+Tools like My GitHub Resume~\cite{my-github-resume} and Visual Resume~\cite{sarma2016}
+that aggregate information from previous contributions can help with this assessment,
+while the OpenHatch project~\cite{openhatch} aggregated entry-level issues from a variety of open source projects
```
Comment (batpigandme) [inline revised.tex]:
```suggestion
while the OpenHatch project~\cite{openhatch} aggregates entry-level issues from a variety of open source projects
```

File: revised.tex [hunk 3]
```diff
@@ -0,0 +1,555 @@
...
+and classified them according to language and other required skills
```
Comment (batpigandme) [inline revised.tex]:
```suggestion
and classifies them according to language and other required skills
```

Comment (batpigandme) [reply revised.tex]: Matching tense to the first part of the first part of the sentence.

File: revised.tex [hunk 4]
```diff
@@ -0,0 +1,555 @@
...
+  or
+
+\item
+  made a significant contribution to lesson development, infrastructure, or other activities
+  as determined by the Executive Council.
+
+\end{itemize}
+
+\noindent
+Decisions are then made by the elected,
```
Comment (batpigandme) [inline revised.tex]:
```suggestion
Decisions are then made by those elected,
```

File: revised.tex [hunk 5]
```diff
@@ -0,0 +1,555 @@
...
+A single project may use wikis, files in GitHub, shared Google Docs, old tweets or Slack messages, and email archives;
+keeping information about a specific topic in a single place
+and clearly defining the purpose of each communication medium
+saves newcomers from having to navigate multiple unfamiliar data sources to find what they need.
+Doing this makes newcomers more confident and oriented~\cite{steinmacher2016}.
+
+At the same time,
+outdated documentation may lead newcomers to a wrong understanding of the project,
+which is also demotivating.
+While it may be hard to keep material up-to-date,
```
Comment (batpigandme) [inline revised.tex]:
```suggestion
While it may be hard to keep material up to date,
```

Comment (batpigandme) [reply revised.tex]: "up to date" is one of those weird modifiers that only gets hyphens when it's before the noun (up-to-date material vs material up to date)

File: revised.tex [hunk 6]
```diff
@@ -0,0 +1,555 @@
...
+Any member of the committee implicated in the incident should be refused from reviewing the violations.
+
+Project leaders should also develop and publicize enforcement mechanisms,
+which may range from verbal or written warnings,
+limits on access to project communication avenues (e.g., Slack channels or mailing lists),
+or suspension or expulsion from contributing to the project.
+When safe for the reporter,
+project leaders should also publicize enforcement decisions:
+if this is not done,
+the community may come to believe that the code as meaningless.
```
Comment (batpigandme) [inline revised.tex]:
```suggestion
the community may come to believe that the code is meaningless.
```

File: revised.tex [hunk 7]
```diff
@@ -0,0 +1,555 @@
...
+People in open source sometimes joke that
+a programmer is someone who will do something for a laptop sticker
+that they would not do for a hundred dollars.
+The kernel of truth in this joke is that
+gratitude and recognition are the most powerful tools community builders have.
+It is therefore crucial to acknowledge newcomers' contributions and thank them for their work.
+Every hour that someone has given your project may be an hour taken away from their personal life
+or their official employment;
+recognize that fact
+and make it clear that while more hours would be welcome,
```
Comment (batpigandme) [inline revised.tex]:
```suggestion
and make it clear that, while more hours would be welcome,
```

File: revised.tex [hunk 8]
```diff
@@ -0,0 +1,555 @@
...
+Who holds the copyright on contributed material?
+Most projects now place this information in files called \texttt{LICENSE.md} and \texttt{CITATION.md},
+and place a brief, readable summary in plain language in onboarding materials.
+
+\rulemajor{Rule 10: Follow up on both success and failure.}
+
+Once someone has carried their first contribution over the line,
+you and they are likely to have a better sense of what they have to offer
+and how the project can help them.
+Helping newcomers finding the next problem they might want to work on
```
Comment (batpigandme) [inline revised.tex]:
```suggestion
Helping newcomers find the next problem they might want to work on
```

**Review by batpigandme — APPROVED** [review]
Looks good. Just copy edits, here.

File: revised.tex (hunk 1 above)
Comment (gvwilson) [reply revised.tex]: :+1:

File: revised.tex (hunk 2 above)
Comment (gvwilson) [reply revised.tex]: Sadly now defunct - note added.

File: revised.tex (hunk 3 above)
Comment (gvwilson) [reply revised.tex]: See above.

File: revised.tex (hunk 5 above)
Comment (gvwilson) [reply revised.tex]: :+1:

File: revised.tex (hunk 6 above)
Comment (gvwilson) [reply revised.tex]: :+1:

File: revised.tex (hunk 8 above)
Comment (gvwilson) [reply revised.tex]: :+1:

File: revised.tex (hunk 7 above)
Comment (gvwilson) [reply revised.tex]: I'm gonna disagree on this one :-)

File: revised.tex (hunk 4 above)
Comment (gvwilson) [reply revised.tex]: :+1:

**Pull request merged**